test/e2e/basic_test.go

```diff
@@ -400,6 +400,9 @@ func testReconcileGatewayRoutesOnStartup(t *testing.T, data *TestData, isIPv6 bo
 				continue
 			}
 			route := Route{}
+			if net.ParseIP(matches[1]) != nil {
+				matches[1] = fmt.Sprintf("%s/32", matches[1])
+			}
```

Comment:
I removed it in this PR. I think in Jenkins or Kind testbed, this is not needed. When there is a routing entry like the following:
```
192.168.10.1 via 169.254.0.253 dev antrea-gw0
```
The routing entry is for existing ServiceLoadBalancer. Since the destination IP is not a CIDR, then the test `TestReconcileGatewayRoutesOnStartup` will fail without this piece of code.